__tests__/Hospital-Detail /doctor/DoctorsList.test.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DoctorsList from '@/components/doctor/DoctorsList';
+import { hospitalsResponse } from './../../mocks/hospitals'; 
+
+jest.mock('@/store/features/doctors/doctors-api', () => ({
```

Comment:
I don't think This mock's response value is hospitalsResponse, rather it is of type FilteredDoctorsResponse. try to look into the response values as this can cause an issue on your test.